lib/clients/authenticated.js

```diff
@@ -111,17 +111,24 @@ _.assign(AuthenticatedClient.prototype, new function() {
   prototype._placeOrder = function(params, callback) {
     var self = this;
 
-    var requiredParams = ['size', 'side', 'product_id'];
+    var requiredParams = ['side', 'product_id'];
 
     if (params.type !== 'market') {
       requiredParams.push('price');
+      requiredParams.push('size');
```

Comment:
In addition to a `funds` parameter? My understanding according to the API reference is that size will be used to place the order but it'll limit the buy according to funds.

>When placing a market order you can specify funds and/or size. Funds will limit how much of your quote currency account balance is used and size will limit the bitcoin amount transacted.